cmd/state-svc/internal/hash/file_hasher.go

```diff
@@ -24,11 +25,11 @@ type FileHasher struct {
 
 func NewFileHasher() *FileHasher {
 	return &FileHasher{
-		cache: cache.New(cache.NoExpiration, cache.NoExpiration),
+		cache: cache.New(cache.NoExpiration, 24*time.Hour),
```

Comment:
```suggestion
		cache: cache.New(24*time.Hour, 24*time.Hour),
```

Otherwise the schedule would run every 24hr, but it would then find there is no expiration so it wouldn't actually expire anything.

Why these are separate inputs is beyond me.